engines/bastion_katello/app/assets/javascripts/bastion_katello/environments/details/views/environment-details.html

```diff
@@ -1,30 +1,28 @@
 <span page-title ng-model="environment">{{ 'Lifecycle Environment:' | translate }} {{ environment.name }}</span>
 
-<div data-extend-template="layouts/two-column-details.html">
```

Comment:
Not required, no. I changed it for demo last week because the page as is was unreadable. It's vaguely related to the following issue, though I changed this to fix the registry name pattern: http://projects.theforeman.org/issues/23494

I can remove into it's own tiny PR, up to you. I'm not ready to touch all the descriptions yet.

Reply:
Yes please. :)